problems/1.types+operators.js

```diff
@@ -2,15 +2,17 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  (val % 2 === 0) || (val % 2 === 0 )
```

Comment:
I'm not sure what you're trying to do with this line. Always use lines to help. Save as a variable, put in conditionals, or return.